index.js

```diff
@@ -1,33 +1,15 @@
-module.exports = function write(stream, chunk, cb){
-  var errored = false;
+const { once } = require('events')
 
-  stream.once('error', error);
-  function error(err){
-    errored = true;
-    cb(err);
-  }
-
-  if (stream.socket && !stream.socket.writable) {
-    var err = new Error('write after end');
+module.exports = async function write(stream, chunk){
+  if (!stream.writable) {
+    var err = new Error('Stream is not writable');
     err.status = 200;
```

Comment:
I don't recall really :/

Reply:
It must have something to do with http frameworks like express/connect/koa, who read `err.status` for constructing an error page. But not sure why it would be 200 🤔  Let's remove it